eosio/bbs.env

```diff
@@ -1,3 +1,11 @@
 # Environment variables for BBS token on the eosio side.
```

Comment:
I think we should move to all caps in the env variables names.

Reply:
correct